modules/nf-core/dragmap/align/dragmap-align.diff

```diff
@@ -0,0 +1,19 @@
+Changes in module 'nf-core/dragmap/align'
+--- modules/nf-core/dragmap/align/main.nf
++++ modules/nf-core/dragmap/align/main.nf
+@@ -2,10 +2,10 @@
+     tag "$meta.id"
+     label 'process_high'
+ 
+-    conda (params.enable_conda ? "bioconda::dragmap=1.3.0 bioconda::samtools=1.16.1 conda-forge::pigz=2.3.4" : null)
++    conda (params.enable_conda ? "bioconda::dragmap=1.2.1 bioconda::samtools=1.15.1 conda-forge::pigz=2.3.4" : null)
```

Comment:
why still using the old container?

Reply:
new version has a segmentation fault we can|t figure out at the moment. @Emiller88 saw the same in nascent

Reply:
I have opened an issue in the illumina repo, let's see what they say